tests/test_loaders_velociraptor.py

```diff
@@ -79,21 +78,117 @@ def test_velociraptor_loader_windows_ntfs(paths: list[str], mock_target: Target,
 @pytest.mark.parametrize(
     "paths",
     [
-        (["uploads.json", "uploads/file/etc", "uploads/file/var"]),
-        (["uploads.json", "uploads/auto/etc", "uploads/auto/var"]),
-        (["uploads.json", "uploads/file/etc", "uploads/file/var", "uploads/file/opt"]),
-        (["uploads.json", "uploads/auto/etc", "uploads/auto/var", "uploads/auto/opt"]),
-        (["uploads.json", "uploads/file/Library", "uploads/file/Applications"]),
-        (["uploads.json", "uploads/auto/Library", "uploads/auto/Applications"]),
+        (
+            [
+                "uploads/mft/%5C%5C.%5CC%3A/",
+                "uploads/mft/%5C%5C.%5CC%3A/$Extend",
+                "uploads/mft/%5C%5C.%5CC%3A/windows/system32",
+                "uploads/mft/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
+            ]
+        ),
+        (
+            [
+                "uploads/ntfs/%5C%5C.%5CC%3A/",
+                "uploads/ntfs/%5C%5C.%5CC%3A/$Extend",
+                "uploads/ntfs/%5C%5C.%5CC%3A/windows/system32",
+                "uploads/ntfs/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
+            ]
+        ),
+        (
+            [
+                "uploads/ntfs_vss/%5C%5C.%5CC%3A/",
+                "uploads/ntfs_vss/%5C%5C.%5CC%3A/$Extend",
+                "uploads/ntfs_vss/%5C%5C.%5CC%3A/windows/system32",
+                "uploads/ntfs_vss/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
+            ]
+        ),
+        (
+            [
+                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/",
+                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/$Extend",
+                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/windows/system32",
+                "uploads/lazy_ntfs/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
+            ]
+        ),
+    ],
+)
+def test_velociraptor_loader_windows_ntfs_zip(paths: list[str], mock_target: Target, tmp_path: Path) -> None:
```

Comment:
I would suggest to add the following helper function at the beginning of the file:

```
def create_paths(sub_dir: str) -> list[str]:
    return [
        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/",
        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/$Extend",
        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/windows/system32",
        f"uploads/{sub_dir}/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
    ]
```

And then you can do the following for both the `test_velociraptor_loader_windows_ntfs` and `test_velociraptor_loader_windows_ntfs_zip` test:    

```
@pytest.mark.parametrize(
    "sub_dir", 
    ["mft", "ntfs", "ntfs_vss", "lazy_ntfs"],
)
def test_velociraptor_loader_windows_ntfs(sub_dir: str, mock_target: Target, tmp_path: Path) -> None:
    paths = create_paths(sub_dir)
```

```
@pytest.mark.parametrize(
    "sub_dir", 
    ["mft", "ntfs", "ntfs_vss", "lazy_ntfs"],
)
def test_velociraptor_loader_windows_ntfs_zip(sub_dir: str, mock_target: Target, tmp_path: Path) -> None:
    paths = create_paths(sub_dir)
```

This reduces the amount of lines and makes it more readable in my opinion :)

Reply:
Would also be nice to do it on a `class` level but I believe that the way the testing is setup that's not supported, see: https://docs.pytest.org/en/7.3.x/how-to/parametrize.html

Reply:
I updated the testing strategy as you suggested!